### PR TITLE
Fix: Add label for empty string in column header

### PR DESCRIPTION
**Pull request by djbarnwal**
If one of the values of a dimension is "" (empty string), the app would crash. This PR adds a label `<empty string>` for such cases.

### CODE REVIEW

File: web-common/src/features/dashboards/pivot/PivotExpandableCell.svelte [hunk 1]
```diff
@@ -22,7 +22,14 @@
       </div>
     </button>
   {/if}
-  {value === "LOADING_CELL" ? "" : value}
+
+  {#if value === "LOADING_CELL"}
+    {""}
+  {:else if value === ""}
+    {"<empty string>"}
```
Comment (pjain1) [inline web-common/src/features/dashboards/pivot/PivotExpandableCell.svelte:29]: is it not possible to display just blank instead of printing this ?

Comment (djbarnwal) [reply web-common/src/features/dashboards/pivot/PivotExpandableCell.svelte:29]: Displaying blank string messes up the styling and also is not supported by the table framework we use. `null` is already a reserve keyword so went with `<empty string>` which is adopted by some browser and tools when you print an empty string.

Comment (djbarnwal) [reply web-common/src/features/dashboards/pivot/PivotExpandableCell.svelte:29]: @mindspank thoughts?

Comment (pjain1) [reply web-common/src/features/dashboards/pivot/PivotExpandableCell.svelte:29]: may be I can merge for now and product can decide correct semantics here

Comment (djbarnwal) [reply web-common/src/features/dashboards/pivot/PivotExpandableCell.svelte]: Switched to NBSP unicode character after discussing with Alex

**Review by pjain1 — APPROVED** [review]

**Pull request merged**